### PR TITLE
gh #8 dsAudio Test Specification Documentation

### PR DESCRIPTION
**Pull request by bhanucbp**
Initial draft verion

### CODE REVIEW

**Review by Ulrond — DISMISSED** [review]
generally looks ok, just some small comments

File: profiles/include/sink/Panel_AudioSettings.yaml [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+    # dsAUDIOPORT_TYPE_SPEAKER     = 0x03,  ///< SPEAKER audio output            
+    # dsAUDIOPORT_TYPE_HDMI_ARC    = 0x04,  ///< HDMI ARC/EARC audio output      
+    # dsAUDIOPORT_TYPE_HEADPHONE   = 0x05,  ///< Headphone jack 
+    # dsAUDIOPORT_TYPE_MAX         = 0x06   ///< Out of range     
+    Typeid: 0x03  # SPEAKER
+    # SPEAKER
+    # HDMI_ARC
+    # SPDIF
+    # HEADPHONE
+    Name: "SPEAKER"
```
[the remark this answers is not in the document]
Comment (Anbukannadhasan) [reply profiles/include/sink/Panel_AudioSettings.yaml:48]: For all ports, added the general information about the all port names

File: profiles/include/sink/Panel_AudioSettings.yaml [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+
+# ID_LR                      
+# HDMI                       
+# SPDIF                    
+# SPEAKER                 
+# HDMI_ARC          
+# HEADPHONE    
+# MAX              
+PortTypes:
+  - SPEAKER
```
[the remark this answers is not in the document]
Comment (Anbukannadhasan) [reply profiles/include/sink/Panel_AudioSettings.yaml]: Modified based on the enum and enum numbers. Done

File: profiles/include/sink/Panel_AudioSettings.yaml [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+# dsAUDIOSUPPORT_NONE             = 0x0,     ///< None
+# dsAUDIOSUPPORT_ATMOS            = 0x01,    ///< Dolby Atmos
+# dsAUDIOSUPPORT_DD               = 0x02,    ///< Dolby Digitial
+# dsAUDIOSUPPORT_DDPLUS           = 0x04,    ///< Dolby Digital Plus
+# dsAUDIOSUPPORT_DAD              = 0x08,    ///< Digital Audio Delivery
+# dsAUDIOSUPPORT_DAPv2            = 0x10,    ///< Digital Audio Processing version 2
+# dsAUDIOSUPPORT_MS12             = 0x20,    ///< Multi Stream 12
+# dsAUDIOSUPPORT_MS12V2           = 0x40,    ///< Multi Stream Version 2
+# dsAUDIOSUPPORT_Invalid          = 0x80,    ///< Invalid
+Audio_Capabilities: 0x37  # dsAUDIOSUPPORT_ATMOS | dsAUDIOSUPPORT_DAPv2 | dsAUDIOSUPPORT_MS12 | dsAUDIOSUPPORT_DD | dsAUDIOSUPPORT_DDPLUS
```
[the remark this answers is not in the document]
Comment (Anbukannadhasan) [reply profiles/include/sink/Panel_AudioSettings.yaml]: Yes, its a global. Moved before ports. Done

**Review by Anbukannadhasan — COMMENTED** [review]
Updated the review comments

**Comment by bhanucbp** [comment]
duplicate of PR #71 , issue #29

**Pull request closed without merging**